composer.json

```diff
@@ -15,15 +15,15 @@
     ],
 
     "require": {
-        "php":                  ">=5.3.6",
+        "php":                  ">=5.5.9",
         "behat/mink":           "^1.7.1@dev",
-        "symfony/browser-kit":  "~2.3|~3.0|~4.0",
-        "symfony/dom-crawler":  "~2.3|~3.0|~4.0"
```

Comment:
Why `"symfony/dom-crawler":  "~2.3|~3.0|~4.0"` line was removed?

Reply:
`dom-crawler` is required by`browser-kit` and no required in code base then this package is not necessary.

Reply:
We do work with the DomCrawler API direclty a lot, even through we don't instantiate the crawler ourselves. Compatibility with dom-crawler versions is very important for this driver.